Cubical/Categories/Displayed/Instances/Presheaf/Base.agda

```diff
@@ -0,0 +1,103 @@
+{-# OPTIONS --safe #-}
+{-# OPTIONS --lossy-unification #-}
+module Cubical.Categories.Displayed.Instances.Presheaf.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Function
+open import Cubical.Data.Sigma
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Presheaf
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Constructions.Elements
+
+open import Cubical.Categories.Displayed.Base
+
+open Category
+open Functor
+open NatTrans
+open Categoryᴰ
+open Contravariant
+
+private
+  variable ℓC ℓC' ℓD ℓD' ℓE ℓE' : Level
+
+module _ (C : Category ℓC ℓC') (ℓSET ℓSETᴰ : Level) where
+  module _ (P : Presheaf C ℓSET) where
+    -- TODO: this is a name clash with the other Presheafᴰ, which is different
+    -- (and not a displayed presheaf in the strictest sense of "displayed X")
+    Presheafᴰ : Type _
+    Presheafᴰ = Presheaf (∫ᴾ P) ℓSETᴰ
```

Comment:
+ we need to figure out what to name this notion of displayed presheaf, which is an instance of the already-existing one, although I'm not sure what the definitional differences are, if any.

Reply:
Hm good question. This one is a "dependent" presheaf maybe? Would it be so bad to still call this `Presheafᴰ` ? I'm not sure

Reply:
I've left the name as-is. At least I'm fine with it.